Lib/test/test_ordered_dict.py

```diff
@@ -507,28 +507,6 @@ def blocking_hash(hash):
         del od[colliding]
         self.assertEqual(list(od.items()), [(key, ...), ('after', ...)])
 
-    def test_issue24347(self):
```

Comment:
These tests test former bugs in C implementation. They should be kept for C implementation for ensuring that these bugs are not reappeared.